Sources/StreamChatSwiftUI/DependencyInjection.swift

```diff
@@ -19,13 +19,13 @@ public struct InjectedValues {
     private static var current = InjectedValues()
     
     /// A static subscript for updating the `currentValue` of `InjectionKey` instances.
-    static subscript<K>(key: K.Type) -> K.Value where K: InjectionKey {
+    public static subscript<K>(key: K.Type) -> K.Value where K: InjectionKey {
```

Comment:
Why do we need to make these two `public`?

Reply:
To allow custom types from integration projects to use the DI. Check the `TwitchInjectedValues` in the clone for more details.

Reply:
ah yes, thanks! Makes sense!